src/components/Card.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { fireEvent } from "@testing-library/react";
+import { noop } from "src/utils";
+import { click, render } from "src/utils/rtl";
+import { Card, CardProps } from "./Card";
+
+describe("Card Component", () => {
+  const defaultProps = {
+    title: "Test Card",
+    subtitle: "Test Subtitle",
+    imgSrc: "plan-exterior.png",
+    detailContent: <div>Detail Content</div>,
+    tag: { text: "Active", type: "success" },
+  } as CardProps;
```

Comment:
nit/suggestion: this may be a better place for `satisfies` vs. casting with `as` since I believe `as` won't error if there are invalid keys the way these options would:

```ts
  const defaultProps = {
    title: "Test Card",
    subtitle: "Test Subtitle",
    imgSrc: "plan-exterior.png",
    detailContent: <div>Detail Content</div>,
    tag: { text: "Active", type: "success" },
  } satisfies CardProps;

// alternatively
  const defaultProps: CardProps = {
    title: "Test Card",
    subtitle: "Test Subtitle",
    imgSrc: "plan-exterior.png",
    detailContent: <div>Detail Content</div>,
    tag: { text: "Active", type: "success" },
  };
```